WordPress/Classes/ViewRelated/Reader/ReaderPostCardCell.swift

```diff
@@ -331,7 +331,10 @@ fileprivate func < <T: Comparable>(lhs: T?, rhs: T?) -> Bool {
         }
 
         featuredImageView.layoutIfNeeded()
-        if featuredImageView.image == nil || featuredImageDesiredWidth != featuredImageView.frame.size.width || featuredImageURL.absoluteString != currentLoadedCardImageURL {
+        if (!featuredImageURL.isGif && featuredImageView.image == nil) ||
+            (featuredImageURL.isGif && featuredImageView.animationImages == nil) ||
```

Comment:
One small detail:
As far as I understand, `featuredImageView.animationImages == nil` will always `true`.
It's probably enough to send the image to reload directly if `featuredImageURL.isGif`.